investpy/__init__.py

```diff
@@ -3161,9 +3370,588 @@ def search_indices(by, value):
     search_result = indices.loc[indices['matches'] == True].copy()
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.